docker/benchmark-builder/Dockerfile

```diff
@@ -11,51 +11,47 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-# Dockerfile for building a specific benchmark for a specific fuzzer.
-#
-# The benchmark/fuzzer pair is defined by build arguments. To specify them, pass
-# the following arguments to docker build:
-#
-# $ docker build \
-#   --build-arg benchmark=afl \
-#   --build-arg fuzzer=freetype2-2017 \
-#   ...
 
-# Start from $fuzzer's builder image.
-ARG fuzzer
-FROM gcr.io/fuzzbench/builders/$fuzzer
+ARG parent_image
+
+# Using multi-stage build to copy latest Python 3.
+FROM gcr.io/fuzzbench/base-image AS base-image
+
+FROM $parent_image
 
-# An ARG declared before a FROM is outside of a build stage scope, so we need to
-# re-declare it here.
 ARG fuzzer
 ARG benchmark
-
-# Set up source and output directories.
-ENV SRC /src
-ENV OUT /out
+ARG checkout_commit
 
 ENV FUZZER $fuzzer
 ENV BENCHMARK $benchmark
-ENV FUZZ_TARGET $OUT/fuzz-target
 
-WORKDIR $SRC
+# Copy latest python3 from base-image into local.
```

Comment:
Can we avoid this somehow, why not inherit from a base- image that has python.

Reply:
I can't really think of a way to avoid this. But if you have any suggestions I'm open to them.
But I think this PR's purpose is to make every benchmark build like an OSS-Fuzz benchmark (which provides the benefit of reducing complexity). I think improving the OSS-Fuzz benchmark build process is a bit out of scope, even if it would be worth while to do another time.